src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -546,9 +551,9 @@ the access legs used. In other cases where the access(CAR) is faster than transi
           .asEnumMap(StreetMode.class, TimeAndCostPenaltyMapper::map)
       )
       .withMaxAccessEgressDuration(
-        c
-          .of("maxAccessEgressDuration")
-          .since(V2_1)
+        accessEgress
+          .of("maxDuration")
+          .since(V2_4)
```

Comment:
We decided a while ago that when we rename config vars we keep the old version since that is when the "concept" has been introduced which is what we want to communicate.

Reply:
ok thanks! I was wondering what to do with the versions, now I know.